packages/lesswrong/server/manualMigrations/index.ts

```diff
@@ -120,3 +120,4 @@ import './2023-02-10-fillCreatedAtForDebouncerEvents'
 import './2023-03-16-createUserActivities'
 import './2023-04-10-importACXmeetups'
 import './2023-06-13-renameShortformToQuicktakes'
+import './2023-07-28-viewUpdaterErrorHandling'
```

Comment:
Can you add a comment to `packages/lesswrong/server/manualMigrations/2023-01-13-createViewUpdater.ts` saying "not the most recent version" or similar?